lib/util/fs.js

```diff
@@ -31,11 +31,42 @@ function ensureDirectoryExists(dir) {
   return stat(dir).catch(() => mkdirp(dir));
 }
 
+/**
+ * Read recursively a directory,
+ * sorting folders and files by alphabetically order
+ *
+ * @param {string} dir
+ * The directory to recursively analyse
+ */
+async function getFilepathsInFolderRecursively(dir) {
+  let results = [];
```

Comment:
looks like it can be `const`

Reply:
we reaffect results with the `concat` later 

https://github.com/knex/knex/pull/3967/files/f5c3c9a0f79271b265ec3f4519b3acda3c68f01f#diff-38843cf9c53f32dffeb2e781e43b4d2fR52

in this case I need a `let` variable ?

Reply:
Oh, I see. It can be quite confusing for other developers when you reassign the results list, though.

Do we need to do it in a mutable way? Wouldn't destructuring work here?
`arr1.push(...arr2) // Adds arr2 items to end of array`